Fix Youtube Link URL

justaashu opened the pull request:
I was using the theme and found out that my YouTube link was not working, the reason was that the theme had prefix `https://youtube.com/c/` which seems to be changed by YouTube, now it's `https://www.youtube.com/channel/`. I have tested it to be working on my setup.

justaashu committed:
Fix Youtube Link URL

LordMathis commented:
Hmm, that's interesting. The /c/ works for me just fine but /channel/ doesn't and returns 404

justaashu committed:
Add support for multiple url paths for YouTube

justaashu commented:
Turns out depending on your channel configuration there are 3 possible urls, e.g.
1. youtube.com/channel/UCUZHFZ9jIKrLroW8LcyJEQQ
2. youtube.com/c/YouTubeCreators
3. youtube.com/user/partnersupport

Reference: https://support.google.com/youtube/answer/6180214?hl=en

I have updated this to make user add full path after youtube.com in config and added a comment for the same in the README
